### PR TITLE
Remove broken inreplace in fsh.

### PR DESCRIPTION
**Pull request by geoff-codes**
~~Pretty much what it says.~~

@mikemcquaid @tdsmith Removed the inreplace block entirely.

### CODE REVIEW

**Comment by geoff-codes** [comment]
What is the meaning of this error?
http://bot.brew.sh/job/Homebrew%20Pull%20Requests/21677/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/pull___clean_https___github_com_Homebrew_homebrew_pull_37023/

**Comment by mikemcquaid** [comment]
@geoff-codes There was an old Git error on the bot that I've just cleaned up.

File: Library/Formula/fsh.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def install
     cd bin do
       if Formula['python'].installed?
         inreplace ["fsh", "fshd", "in.fshd"],
-            "#! /usr/local/bin/python", "#!/usr/bin/env python"
+            "#! /usr/bin/python", "#!/usr/bin/env python"
       end
```
Comment (mikemcquaid) [inline Library/Formula/fsh.rb]: We may want to get rid of this `inreplace` altogether now CC @tdsmith

Comment (tdsmith) [inline Library/Formula/fsh.rb]: agreed

File: Library/Formula/fsh.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@ def install
     cd bin do
```
Comment (geoff-codes) [inline Library/Formula/fsh.rb]: @mikemcquaid @tdsmith So axe this block?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `fsh: cleanup, remove bad inreplace and add test.`.

**Pull request closed without merging**